components/site/docsHomePageContent.tsx

```diff
@@ -0,0 +1,374 @@
+  desc2: (
+    <>
+      To find the best resource for your question and fastest resolution, check out our
+      <Link href="/" passHref>
+        <SimpleStyledLink $inlineBlock={true}>how to get help guide</SimpleStyledLink>
+      </Link>
+    </>
+  ),
+  content: [
+    { text: 'Submit a ticket', url: '/submit-a-ticket', Icon: Feedback },
+    { text: 'FAQs', url: '/', Icon: HelpOutline },
+    { text: 'Slack Community', url: '/', Icon: QuestionAnswer },
+  ],
+};
+
+/*
+ ██████╗ ██████╗ ███╗   ███╗██████╗  ██████╗ ███╗   ██╗███████╗███╗   ██╗████████╗███████╗
+██╔════╝██╔═══██╗████╗ ████║██╔══██╗██╔═══██╗████╗  ██║██╔════╝████╗  ██║╚══██╔══╝██╔════╝
+██║     ██║   ██║██╔████╔██║██████╔╝██║   ██║██╔██╗ ██║█████╗  ██╔██╗ ██║   ██║   ███████╗
+██║     ██║   ██║██║╚██╔╝██║██╔═══╝ ██║   ██║██║╚██╗██║██╔══╝  ██║╚██╗██║   ██║   ╚════██║
+╚██████╗╚██████╔╝██║ ╚═╝ ██║██║     ╚██████╔╝██║ ╚████║███████╗██║ ╚████║   ██║   ███████║
+ ╚═════╝ ╚═════╝ ╚═╝     ╚═╝╚═╝      ╚═════╝ ╚═╝  ╚═══╝╚══════╝╚═╝  ╚═══╝   ╚═╝   ╚══════╝
+*/
+
+const SectionContent = (props: SectionContentProps): JSX.Element => {
+  const { title, desc, desc2 } = props;
+  return (
+    <>
+      <Text fontWeight="semibold" fontSize="500" lineHeight="500">
+        {title}
+      </Text>
+      <Text my="300" mb={!desc2 ? '550' : '300'}>
+        {desc}
+      </Text>
+      {desc2 && <Text mb="550">{desc2}</Text>}
+    </>
+  );
+};
+
+const GuideLink = (props: GuideLinkProps) => {
+  const { url, text } = props;
+  return (
+    <Link href={url} passHref>
+      <SimpleStyledLink>{text}</SimpleStyledLink>
+    </Link>
+  );
+};
+
+const Card = (props: CardProps): JSX.Element => {
+  const { url, title, desc, linkText } = props;
+  return (
+    <Link href={url} passHref>
+      <StyledCard
+        p="16px 20px"
+        borderWidth="1px"
+        borderStyle="solid"
+        borderColor="gray.300"
+        width={['100%', '48%', '31%']}
+        position="relative"
+        minWidth="175px"
+        my={['500', null, '200']}
+        display="flex"
+        flexDirection="column"
+        justifyContent="space-between"
+      >
+        <Box fontWeight="semibold" fontSize="400" lineHeight="400">
+          {title}
+        </Box>
+        <Box fontSize="300" lineHeight="300" my="200">
+          {desc}
+        </Box>
+        <SimpleStyledLink>{linkText}</SimpleStyledLink>
+      </StyledCard>
+    </Link>
+  );
+};
+
+const Library = (props: LibraryProps): JSX.Element => {
+  const { Icon, name, url } = props;
+  return (
+    <Link href={url} passHref>
+      <BoxStyledLink>
+        <Icon /> {name}
+      </BoxStyledLink>
+    </Link>
+  );
+};
+
+const HelpLink = (props: HelpLinkProps): JSX.Element => {
+  const { Icon, text, url } = props;
+  return (
+    <Link href={url} passHref>
+      <BlueStyledLink>
+        <Icon /> {text}
+      </BlueStyledLink>
+    </Link>
+  );
+};
+
+const DocsHomePageContent = () => {
+  return (
+    <>
+      <Stack>
+        <Box p="800">
```

Comment:
There seems to be some excessive padding at the bottom of this panel, not sure where its coming from though

See bottom of this screen shot
![Screen Shot 2021-11-17 at 8 14 22 AM](https://user-images.githubusercontent.com/3903325/142207512-83e02b6d-d78e-4a31-b8da-ed1b7350ead1.png)

Reply:
HA! Way too much (Dumb and Dumber popped into my head - "Samsonite! I was way off!) :-D 

New commit inbound.